demisto_sdk/commands/init/tests/initiator_test.py

```diff
@@ -473,6 +475,60 @@ def test_template_integration_init(monkeypatch, initiator, tmpdir, template):
     assert not diff, f'There\'s a missing file in the copied files, diff is {diff}'
 
 
+def test_integration_init_with_ignore_secrets(initiator, tmpdir, monkeypatch, mocker):
+    """
+    Tests `integration_init` function with a given script template name.
+    Given
+        - An Integration template - HelloWorld.
+    When
+        - Running the init command with secrets ignore.
+    Then
+        - Ensure the function's ('integration_init') return value is True
+        - Ensure integration directory with the desired integration name is created successfully.
+        - Ensure integration directory contain all files.
+        - Ensure .secrets-ignore file is not empty.
+    """
+    monkeypatch.setattr('builtins.input', generate_multiple_inputs(deque(['6.0.0', 'y'])))
+    white_list = {
+        "files": [],
+        "iocs": {},
+        "generic_strings": []
+    }
+    temp_test_dir = os.path.join(tmpdir, 'Tests')
+    os.makedirs(temp_test_dir, exist_ok=True)
+    with open(f'{temp_test_dir}/secrets_white_list.json', 'w') as f:
+        json.dump(white_list, f)
+
+    temp_pack_dir = os.path.join(tmpdir, 'Packs', PACK_NAME)
+    os.makedirs(temp_pack_dir, exist_ok=True)
+    secrets_ignore_path = f'{temp_pack_dir}/.secrets-ignore'
+
+    initiator.output = temp_pack_dir
+    initiator.dir_name = INTEGRATION_NAME
+    initiator.is_integration = True
+    initiator.template = 'HelloWorld'
+    initiator.category = 'Utilities'
+    mocker.patch('demisto_sdk.commands.init.initiator.get_pack_name', return_value='PackName')
+
+    integration_path = os.path.join(temp_pack_dir, INTEGRATION_NAME)
+
+    with ChangeCWD(tmpdir):
+        res = initiator.integration_init()
```

Comment:
Where here we are running the init with secrets ignore? Isn't that something the user should insert?